src/main/java/com/napier/sem/ReportLanguageImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.napier.sem;
+
+public class ReportLanguageImpl implements ReportGenerator{
+
+    DatabaseConnection dbConnection;
+
+    public ReportLanguageImpl(DatabaseConnection dbConnection){
+        this.dbConnection = dbConnection;
+    }
+
+    @Override
+    public String generateReport(String sqlStatement){
+
+        return dbConnection.executeSQLStatement(sqlStatement);
+
+        /*
```

Comment:
Should remove comments if the code isnt needed

Reply:
Comment removed.